youtube/templates/watch.html

```diff
@@ -14,6 +14,21 @@
         text-decoration: underline;
     }
 
+    details.replies > summary{
+        background-color: var(--interface-color);
+        border-style: outset;
+        border-width: 1px;
+        font-weight: bold;
+        padding-bottom: 0px;
+    }
+
+    details.replies .comment{
+        width: 600px;
+        border-style: solid;
+        border-width: 1px;
+        border-color: #0402025e;
```

Comment:
The border does more harm than good visually. It looks better without it in my opinion.